Makefile

```diff
@@ -56,6 +59,17 @@ mnesia_backup:
 		-e backup_suffix=$(BACKUP_SUFFIX) \
 		mnesia_backup.yml
 
+~/.ssh/id_ae_infra_ed25519:
+	@ssh-keygen -t ed25519 -N "" -f $@
+
+~/.ssh/id_ae_infra_ed25519-%-cert.pub: ~/.ssh/id_ae_infra_ed25519
+	@vault write -field=signed_key ssh/sign/$* public_key=@$<.pub > $@
```

Comment:
I assume this line is generating locally CA-signed cert from pubkey after having checked on vault remote server that user to whom the GitHub token is associated is authorized. Right?

Reply:
right, you can read more about this method here: https://code.fb.com/security/scalable-and-secure-access-with-ssh/

However, they use their own tooling instead of vault